inc/task.class.php

```diff
@@ -1625,6 +1631,152 @@ function post_updateItem($history=1) {
 
 
 
+   function csvExport($params) {
+      $includeoldjobs    = $_SESSION['glpi_plugin_fusioninventory']['includeoldjobs'];
+      $agent_state_types = ['prepared', 'cancelled', 'running', 'success', 'error' ];
+      if (isset($params['agent_state_types'])) {
+         $agent_state_types = $params['agent_state_types'];
+      }
+
+      // 0 : no debug (really export to csv,
+      // 1 : display final table,
+      // 2 : also display json
+      define('DEBUG_CSV', 0);
+      if (!DEBUG_CSV) {
```

Comment:
You define `DEBUG_CSV` 2 lines before so I think it's for debug and need to be remove, but see that, I'm not sure it right defined in another part of code